habitat/tasks/rearrange/sub_tasks/nav_to_obj_sensors.py

```diff
@@ -131,9 +140,15 @@ def get_observation(self, task, *args, **kwargs):
 
 
 @registry.register_sensor
-class NavGoalSensor(Sensor):
+class NavGoalSensor(UsesRobotInterface, Sensor):
```

Comment:
Why does this use the `UsesRobotInterface` but does not use self.robot_id? Seems like this inheritance is not useful

Reply:
Good catch. Removed.